client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -172,6 +175,14 @@ def log_model(
         """
         raise NotImplementedError
 
+
+    @classmethod
+    def _validate_artifact_uploaded(cls, artifact_msg):
```

Comment:
Since `cls` goes unused, I'd recommend going with a `staticmethod` in the interesting of being persimmonious.

```suggestion
    @staticmethod
    def _validate_artifact_uploaded(artifact_msg):
```